frontend_csv/src/parser/ast_msg.rs

```diff
@@ -32,38 +33,38 @@ pub struct AstMsg {
     pub msg_val: String,
 }
 
-pub struct InitMsg {
+pub struct Msg {
     pub raw: Vec<(AstMsgLineType, Vec<String>)>,
```

Comment:
```suggestion
    struct RawCsv {
       lineType: AstMsgLineType, 
       values: Vec<String>
    }
    pub raw: Vec<RawCsv>,
```

We can use a struct in this case? also I did not catch why we need it BTW

Reply:
for easy access to value of the message